docs/learn/learn-xcm-transport.md

```diff
@@ -41,8 +41,8 @@ _Direct_ and _Relayed_.
 - With _Direct_, message data goes direct between parachains and is O(1) on the side of the
   Relay-chain and is very scalable.
 - With _Relayed_, message data is passed via the Relay-chain, and piggy-backs over VMP. It is much
-  less scalable, and parathreads in particular may not receive messages due to excessive queue
-  growth.
+  less scalable, and on-demand parachains in particular may not receive messages due to excessive
+  queue growth.
```

Comment:
Curious if this is indeed the case, not sure how queue is managed in relation to on-demand (I suspect it's the same, still good to check probably though)

Reply:
not sure, I did not touch that sentence if not just for swapping parathread <-> on-demand parachain